app/static/js/script.js

```diff
@@ -3,20 +3,18 @@
 	// Initialize appliciation
   	var app = {
 		init: function() {
-			window.onload = () => {
```

Comment:
Because the `script` tag is already at the bottom of the page `window.onload` isn't necessary.